backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/notification/impl/UserNotificationsConfigRepository.java

```diff
@@ -147,4 +197,22 @@ else if (X_AD_User_NotificationGroup.NOTIFICATIONTYPE_NotifyUserInCharge.equals(
 			throw new AdempiereException("Unknown notification type: " + notificationTypeCode);
 		}
 	}
+
+	@NonNull
+	private static String fromNotificationType(@NonNull final NotificationType notificationType)
+	{
+		switch (notificationType)
+		{
+			case Notice:
+				return X_AD_User_NotificationGroup.NOTIFICATIONTYPE_Notice;
+			case EMail:
+				return X_AD_User_NotificationGroup.NOTIFICATIONTYPE_EMail;
+			case NotifyUserInCharge:
+				return X_AD_User_NotificationGroup.NOTIFICATIONTYPE_NotifyUserInCharge;
+			default:
+				//shall never happen
```

Comment:
if shall never happend, then why not throwing exception?

Reply:
done.